atrackerv2/atracker_v2_integration_test.go

```diff
@@ -174,6 +174,31 @@ var _ = Describe(`AtrackerV2 Integration Tests`, func() {
 			fmt.Fprintf(GinkgoWriter, "Saved logdna targetIDLink value: %v\n", targetIDLink)
 		})
 
+		It(`CreateTarget(createTargetOptions *CreateTargetOptions)`, func() {
+
+			eventstreamsEndpointPrototypeModel := &atrackerv2.EventstreamsEndpointPrototype{
+				TargetCRN: core.StringPtr("crn:v1:bluemix:public:messagehub:us-south:a/11111111111111111111111111111111:22222222-2222-2222-2222-222222222222::"),
+				Topic: core.StringPtr("my-test-topic"),
+				Brokers: []string{"kafka-x:9094"},
+				Password: core.StringPtr("xxxxxxxxxxx"),
+			}
+
+			createTargetOptions := &atrackerv2.CreateTargetOptions{
+				Name:           core.StringPtr("my-ies-target"),
+				TargetType:     core.StringPtr("event_streams"),
+				EventstreamsEndpoint: eventstreamsEndpointPrototypeModel,
+			}
+
+			target, response, err := atrackerService.CreateTarget(createTargetOptions)
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(target).ToNot(BeNil())
+
+			targetIDLink2 = *target.ID
```

Comment:
Probably shouldn't share the link variable with the testcase above.
```suggestion
			targetIDLink3 = *target.ID
```